### PR TITLE
Task3_ASP.NET_Core

### PR DESCRIPTION
**Pull request by boOhdan**

### CODE REVIEW

File: ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                _unitOfWork.ProductsRepo.Update(product);
+                _unitOfWork.Commit();
+
+                return RedirectToAction(nameof(Index));
+            }
+
+            return View(product);
+        }
+
+        public IActionResult Delete(int? id)
```
Comment (yevhenbilushko) [inline ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs:100]: add http method attribute

File: ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+            if (product is null)
+            {
+                return NotFound();
+            }
+
+            return View(product);
+        }
+
+        public IActionResult Edit(int? id)
```
Comment (yevhenbilushko) [inline ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs:62]: add http method attribute

File: ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                _unitOfWork.ProductsRepo.Insert(product);
+                _unitOfWork.Commit();
+
+                return RedirectToAction(nameof(Index));
+            }
+
+            return View(product);
+        }
+
+        public IActionResult Details(int? id)
```
Comment (yevhenbilushko) [inline ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs:44]: add http method attribute

File: ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+        {
+            _unitOfWork = unitOfWork;
+        }
+
+        public IActionResult Index()
+        {
+            return View(_unitOfWork.ProductsRepo.Get(includeProperties: "Supplier,ProductCategory"));
+        }
+
+        public IActionResult Create()
```
Comment (yevhenbilushko) [inline ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs:23]: add http method attribute

File: ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    public class ProductsMvcController : Controller
+    {
+        private readonly IUnitOfWork _unitOfWork;
+
+        public ProductsMvcController(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+        }
+
+        public IActionResult Index()
```
Comment (yevhenbilushko) [inline ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs:17]: add http method attribute

File: ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+            if (product is null)
+            {
+                return NotFound();
+            }
+
+            return View(product);
+        }
+
+        [HttpPost]
```
Comment (yevhenbilushko) [inline ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs:117]: [HttpPost]
->
[HttpDelete]

**Review by yevhenbilushko — APPROVED** [review]
fix and merge

File: ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs (hunk 6 above)
Comment (boOhdan) [reply ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs:117]: Browsers only support HTTP GET and HTTP POST verbs when using HTML. This is what I found in StackOverflow

Comment (boOhdan) [reply ASP.NET_Core/FoodOrdering.API/Controllers/ProductsMvcController.cs:117]: https://stackoverflow.com/questions/63244932/how-to-call-an-endpoint-with-httpdelete-attribute-in-asp-net-core-mvc

**Pull request merged**